egs2/talromur/tts1/phonetize.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Could you move this script to local since I want to keep only main scripts under tts1?